lib/APPConvoyLeader/MotorSpeedCalibrationState.cpp

```diff
@@ -218,7 +218,7 @@ void MotorSpeedCalibrationState::finishCalibration(StateMachine& sm)
         LOG_DEBUG_VAL(TAG, "Calibrated max. speed (steps/s): ", maxSpeed);
         LOG_DEBUG_VAL(TAG, "Calibrated max. speed (mm/s): ", maxSpeed / RobotConstants::ENCODER_STEPS_PER_MM);
 
-        sm.setState(&LineSensorsCalibrationState::getInstance());
+        sm.setState(&ReadyState::getInstance());

```

Comment:
Does this mean the line sensors are not used at all or just for now?

Reply:
Just for now. They are not relevant to the application yet, but will be used in the future.
Removing the state altogether would be a waste of time if it has to be brought back at a later point.